backend/substrapp/compute_tasks/outputs.py

```diff
@@ -54,64 +54,50 @@ def _save_performance(self, output: context.OutputResource):
             }
             client.register_performance(performance_obj)
 
-    def _save_models(self, models: list[context.OutputResource]):
-        logger.info("saving models")
+    @retry(raise_exception=True)
+    def _save_model_to_local_storage(self, models: list[context.OutputResource]) -> dict:
+        files = {}
+        metadata = {}
 
-        new_models = []
         for model in models:
-            new_model = self._save_model_to_local_storage(model)
-            new_models.append(new_model)
-
-        try:
-            with get_orchestrator_client(self._ctx.channel_name) as client:
-                registered_models = client.register_models({"models": new_models})
-        except Exception as exc:
-            for model in new_models:
-                self._delete_model(model["key"])
-            raise exc
-
-        for registered_model in registered_models:
-            registered_model["channel"] = self._ctx.channel_name
-            serializer = ModelRepSerializer(data=registered_model)
-            try:
-                serializer.save_if_not_exists()
-            except AlreadyExistsError:
-                pass
-
-        for model, local in zip(models, new_models):
             path = os.path.join(self._ctx.directories.task_dir, model.rel_path)
-            add_model_from_path(path, local["key"])
+            checksum = get_hash(path, self._ctx.task.key)
+
+            files[checksum] = open(path, "rb")
+
+            metadata[checksum] = {
+                "compute_task_key": self._ctx.task.key,
+                "compute_task_output_identifier": model.identifier,
+                "category": _get_model_category(model.identifier),
+            }
 
-    def _save_model_to_local_storage(self, model: context.OutputResource) -> dict:
-        path = os.path.join(self._ctx.directories.task_dir, model.rel_path)
-        from substrapp.models import Model
+        files["metadata"] = json.dumps(metadata)
 
-        checksum = get_hash(path, self._ctx.task.key)
-        instance = Model.objects.create(checksum=checksum)
+        response = organization_client.post(
+            self._ctx.channel_name,
+            settings.LEDGER_MSP_ID,
+            settings.DEFAULT_DOMAIN + reverse("api:model-list"),
```

Comment:
ironically, we still need django just for this part

Reply:
We need django for the reverse but we could hardcode the value.

Reply:
I agree if we consider these are two distinct services, we can handle the API URL using env var, as we probably do for the frontend.

Reply:
As the dependency issue is solved, I guess this is not a big deal for now,